pkg/resource/statefulset.go

```diff
@@ -226,6 +226,10 @@ func (b StatefulSetBuilder) makePodTemplate() corev1.PodTemplateSpec {
 		pod.Spec.Tolerations = b.Spec().Tolerations
 	}
 
+	if utilfeature.DefaultMutableFeatureGate.Enabled(features.TopologySpreadRules) {
```

Comment:
could you also add this to pkg/resource/job.go? https://github.com/cockroachdb/cockroach-operator/blob/master/pkg/resource/job.go#L105

Reply:
Done